README.md

```diff
@@ -381,6 +381,55 @@ const hubspot = new Hubspot(params)
 return hubspot.oauth.getAccessToken(params).then(...)
 ```
 
+## Contributing
+
+Before submitting a pull request, please make sure the following is done:
+
+1. Fork the repository and create your branch from master.
+1. Run npm install in the repository root.
+1. If you’ve fixed a bug or added code that should be tested, add tests!
+1. Ensure the test suite passes (npm test). Tip: `npm run mocha -- --grep "test name"` is helpful in development.
+1. Format your code with prettier and eslint (npm run lint).
```

Comment:
there are a few different commands mentioned in passing here, would it be helpful to have one command to rule them all formatted as a code block?

`npm run build` or `npm run preflight` or whatever the current node standard is.

Reply:
Searching GitHub:

search                                                                    | result count
------------------------------------------------------------------------- | ------------
filename:package.json scripts preflight node extension:json language:JSON | 116
filename:package.json scripts build node extension:json language:JSON     | 376,446
filename:package.json scripts setup node extension:json language:JSON     | 61,568

So I'll go with build. Thanks 👍

Reply:
Done in 
https://github.com/MadKudu/node-hubspot/pull/138/commits/23e467b0b7988439d1a1835c07fa3b4893327127